src/executables/util_cs.c

```diff
@@ -4624,14 +4631,12 @@ memmon (UTIL_FUNCTION_ARG * arg)
       if (module_index < 0 || module_index > MMON_MODULE_LAST)
```

Comment:
```suggestion
      if (module_index < 0 || module_index >= MMON_MODULE_LAST)
```

Reply:
The `module_index` entered by the user has a value 1 greater than the enum value of the module registered in `MMON_MODULE_ID`, so having the same value as `MMON_MODULE_LAST` is valid.

Reply:
I understand.

Nit:
To syncronize the module index entered by user and registered module index in the MMON_MODULE_ID, how about moving last -all option?

Reply:
Regardless of -m all, I believe that starting indexing of specific modules from 1 in the user's perspective is correct for the consistency of cubrid utility(ex) diagdb).

Reply:
I don't agree with you. Based on your opinion -all option should be -1 because diagdb uses -1 for all (-1 dump all). And the module index assigned by you has no specific information. Share your opinion.

Reply:
Before the addition of the `-m all` option, the index of individual modules as seen by the user started from 1, based on the precedent of `diagdb`, which was already agreed. Additionally, in order to synchronize the indexing between the server and the user, there was a proposal to insert a dummy in `MMON_MODULE_ID`. However, due to concerns that inserting a dummy could lead to human errors, the decision was made to remove the dummy. The resulting index discrepancy was resolved by subtracting 1 from the index at server, a conclusion reached during the prior discussion.

Now, suggesting that the index of individual modules as viewed by the user should start from 0 is, in my opinion, a kind of specification change. I am not fully convinced or in agreement to make this change at this point.

Reply:
Done.

Reply:
We already know what you said. I just leave a comment to reduce mis-understanding caused by mis-matching between index inputed by user and index registered internally. And we agree with make two indexes matching off-line.